nomenclature/codelist.py

```diff
@@ -615,17 +615,18 @@ def from_directory(
                         code_list.append(RegionCode(name=c.name, hierarchy="Country"))
 
             # importing from an external repository
-            if config.definitions.region.repository:
+            for repository in config.definitions.region.repositories:
```

Comment:
For consistency with the loop above...
```suggestion
            for repo in config.definitions.region.repositories:
```